specification.md

```diff
@@ -76,20 +77,27 @@ where:
 
 ### Steps
 
+Out of band:
+
+-   Agree on a PAYLOAD_TYPE and cryptographic details.
+-   Decide if [backwards compatible signature] mode should be allowed.
```

Comment:
Once #11 is merged, we probably want to add something here about agreeing how to compute keyid?

Reply:
As long as there aren't duplicate keyids, I don't think the mechanism for computing keyids needs to be known by the client.

Reply:
You're quite right, thanks Marina.

Reply:
Yup, agreed. We'd want to say something like, "Agree on how keyid will be used." On the signer side, this means how it is computed; on the verifier side, it is how the key is looked up given the keyid. We should also explain that the verifier SHOULD try all possible keys that match that keyid.

Reply:
#11 has now been merged, apologies for the delay.

Reply:
Thanks! No delay at all!

My inclination is to merge this as-is and leave further specification to #9. Currently this doesn't have any mention of multiple keys at all, and without multiple keys, keyid doesn't add any value.

Reply:
I agree, let's leave it to #9.